as-vending/res/configuration.yaml

```
@@ -0,0 +1,16 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+Writable:
+  LogLevel: INFO
+
+Service:
+  Host: localhost
+  Port: 48099
+  StartupMsg: This microservice checks if ID numbers from REST requests are authenticated
+
```

core command client isn't in the common config, so needs to be added here
```suggestion
Clients:
    core-command:
      Protocol: "http"
      Host: "localhost"
      Port: 59882
```